mbuild/compound.py

```diff
@@ -3228,6 +3242,58 @@ def from_parmed(self, structure, coords_only=False, infer_hierarchy=True):
             infer_hierarchy=infer_hierarchy,
         )
 
+    def to_rdkit(self):
+        rdkit = import_("rdkit")
+        from rdkit import Chem
+        from rdkit.Chem import AllChem
+
+        for particle in self.particles():
+            if particle.element is None:
+                try:
+                    particle._element = element_from_symbol(particle.name)
+                except ElementError:
+                    try:
+                        particle._element = element_from_name(particle.name)
+                    except ElementError:
+                        raise MBuildError(
+                            "No element assigned to {}; element could not be"
+                            "inferred from particle name {}. Cannot perform"
+                            "an energy minimization.".format(
+                                particle, particle.name
+                            )
+                        )
+
+        temp_mol = Chem.RWMol()
+        p_dict = {particle: i for i, particle in enumerate(self.particles())}
+
+        bo_dict = {
+            1.0: Chem.BondType.SINGLE,
+            2.0: Chem.BondType.DOUBLE,
+            3.0: Chem.BondType.TRIPLE,
+            1.5: Chem.BondType.AROMATIC,
+            0.0: Chem.BondType.UNSPECIFIED,
+            "default": Chem.BondType.SINGLE,
```

Comment:
I think we should raise a single warning here that the `"default"` bond with be converted to single bond in `rdkit`.

Reply:
So, it's not clear to me what the use of having `"default"` is with the changes discussed of how to specify bond order types. Do we need to have `"default"` as an option here, or in `add_bond` if the default is always a single bond?